README.md

```diff
@@ -22,6 +22,7 @@ deployed on the Flow Testnet.
 
 _🛠 This project requires `NodeJS v16.x` or above._ See: [Node installation instructions](https://nodejs.org/en/) <br/>
 _🛠 This project requires `flow-cli v0.32.1` or above._ See: [Flow CLI installation instructions](https://docs.onflow.org/flow-cli)
+_🛠 This project requires `Python v3.6` or above._ See: [Python installation instructions](https://www.python.org/downloads)
```

Comment:
Should we have Python as a "hard" requirement, or can we annotate this to make it clear this is only necessary if you want to run the integration tests yourself, locally? 

Installing Python is not always straightforward, and we want to keep this project as accessible as possible.

Reply:
Python is required on some systems to run npm i when there are no prebuilt binaries for the package you are installing. I ran into this problem with sqlite package on Apple M1.

Reply:
Just ran into this as well

Reply:
Ah yes. SQLite.